build.bash

```diff
@@ -265,7 +269,17 @@ if [ "$hw_platform" == "pine64-xenial" ]; then
   if mv $buildfolder/xenial-pine64-*.img $imagefile; then echo "OK"; else echo "FAILED"; exit 1; fi
 
 # Build Raspberry Pi image
-elif [ "$hw_platform" == "pi-raspbian" ]; then
+elif [ "$hw_platform" == "pi-raspios32" ] || [ "$hw_platform" == "pi-raspios64beta" ]; then
+  if [ "$hw_platform" == "pi-raspios64beta" ]; then
+    zipfile=raspios_lite_arm64_latest.zip
+    baseurl="https://downloads.raspberrypi.org/raspios_arm64/images/raspios_arm64-2020-05-28/2020-05-27-raspios-buster-arm64.zip"
```

Comment:
You should add a note to change the link to a permanent redirect to the latest version when it becomes available.

Reply:
sure. just the one beta image available as of now.